lib/srv/heartbeat.go

```diff
@@ -133,6 +135,8 @@ const (
 	HeartbeatModeApp
 	// HeartbeatModeDB sets heatbeat to db
 	HeartbeatModeDB
+	// HeartbeatModeDatabaseService sets heatbeat mode to Databaseservice.
```

Comment:
nitpick:
```suggestion
	// HeartbeatModeDatabaseService sets heatbeat mode to database service.
```
or `DatabaseService`?

Reply:
`DatabaseService`
I wrote `DatabaseService` everywhere (I think) to ease the grepping for this particular resource type

But can refactor to `database service` if you think it's better

Reply:
Either way is good. As long as it's readable.